### DISCUSSION TITLE
cannot add :after selector on tuiButton

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9913] [Q&A] [answered]
hi, i am trying to add :after selector on tuiButton but it does not appear there. After investigation i find out that ```tuiicons``` attribute is added and if i remove it :after element appears.
How to resolve it?
Thanks

**Comment by waterplea** [comment, marked as answer]
This is expected, `::before` and `::after` are reserved for icons in buttons, links, tabs etc. If you do not use icons and really need to use them for something else — you would have to override the styles that are added by `tuiIcons` directive.

**Comment by vytautas-pranskunas-** [comment]
Thanks
